### PR TITLE
Orc flags

### PR DESCRIPTION
**Pull request by runtd**
## Changelog:
- Added around 30 random coat of arms emblems for orcs.

## Tests:
- [ ] The new CoAs are implemented correctly
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

### CODE REVIEW

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt [hunk 1]
```diff
@@ -761,6 +761,41 @@ colored_emblem_texture_lists = {
 			100 	= "wc_generic_ce_orc_symbol_68.dds"
 			100 	= "wc_generic_ce_orc_symbol_69.dds"
 			100 	= "wc_generic_ce_orc_symbol_70.dds"
+			100 	= "ce_rune_orc_01.dds"
```
Comment (zumbak04) [inline common/coat_of_arms/template_lists/colored_emblem_lists.txt:764]: I would resize this.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt (hunk 1 above)
Comment (runtd) [reply common/coat_of_arms/template_lists/colored_emblem_lists.txt:764]: Resized it.
image

Comment (zumbak04) [inline common/coat_of_arms/template_lists/colored_emblem_lists.txt:764]: There's another range with orcish emblems at 1774 line. So they should be duplicated there too with 2000 chance of appearing.

Duplication isn't good stuff, but sadly it's how Paradox made CoAs.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt (hunk 1 above)
Comment (runtd) [reply common/coat_of_arms/template_lists/colored_emblem_lists.txt:764]: Done !

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**